### DISCUSSION TITLE
Join to project

### DISCUSSION THREAD

**Discussion by Lancetnik** [lwinterface/panini#265] [Show and tell] [answered]
@i2gor87 @artas728 guys, you are really cool! This project is quite interesting, but I suppose, we are working on a pretty close thing. Please, take a look at my [**Propan**](https://github.com/Lancetnik/Propan) project. What are you think about join to me and scaling your ideas to the multiple-brokers framework, not only the Nats one?

**Comment by artas728** [comment, marked as answer]
Thank you for your interest and kind words. However, we have decided to focus solely on Panini for our project and do not plan to join or expand into other projects at this time.

**Reply by Lancetnik** [reply]
Okey, good luck with the Project

**Reply by Lancetnik** [reply]
@artas728 sorry, but I should ping you last time 😄 
What do you think about joing to **FastStream** now? - https://github.com/airtai/faststream
